content/outcomes/application-observability/tsm-posture-health-perf-distributed-app-day1.md

```diff
@@ -0,0 +1,38 @@
+---
+date: '2021-07-13'
+lastmod: '2021-07-13'
+layout: single
+team:
+  - Mark Schweighardt
+  - Niran Even-Chen
+  - Andrew Huffman
+title: "Tanzu Service Mesh: Understand the posture (health and performance) of your distributed application (day 1)"
```

Comment:
I would remove `(day 1)` from the title. It's not necessary. If the reader is interested in the outcome of better understanding health and performance of a distributed app, then they will click the link and read the article. Classifying this as day 1 or day 2 doesn't add much.

Reply:
fixed